### PR TITLE
Sewel Add&App

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: history-of-quakers/modernized/27-addenda.adoc [hunk 1]
```diff
@@ -21,6 +14,10 @@ And the original in Latin, soon after the first publication,
 being translated into English by B. F. was printed in the year 1663,
 from which translation take it as follows:
 
+[.embedded-content-document]
```
Comment (maijapapaya) [inline history-of-quakers/modernized/27-addenda.adoc:17]: I wasn't absolutely certain if this should be embedded or not.

File: history-of-quakers/modernized/28-appendix.adoc [hunk 2]
```diff
@@ -7,9 +7,8 @@ and the plainness of its expressions.
 It is the more like the man,
 and carries the greater evidence of the honesty and integrity of the relater:
 
-The account of the sufferings of Richard Seller,
-of Keinsey a fisherman who was pressed in Scarborough-Piers,
-in the time of the two last engagements between the Dutch and English, in the year 1665.
+[.blurb]
+=== The account of the sufferings of Richard Seller, of Keinsey a fisherman who was pressed in Scarborough-Piers, in the time of the two last engagements between the Dutch and English, in the year 1665.
```
Comment (maijapapaya) [inline history-of-quakers/modernized/28-appendix.adoc:14]: Also, I wasn't sure if this should be embedded, I'm thinking now that it probably should be.

File: history-of-quakers/modernized/28-appendix.adoc [hunk 3]
```diff
@@ -526,10 +525,11 @@ at length became his friends and favorers.
 [.asterism]
 '''
```
Comment (maijapapaya) [inline history-of-quakers/modernized/28-appendix.adoc:531]: Not sure if this is redundant with how the spacing looks, but it was in the original so I left it for you to decide.

File: history-of-quakers/modernized/28-appendix.adoc [hunk 4]
```diff
@@ -729,10 +728,10 @@ he was slow in taking off that where the gold was, and the marshal thinking it t
 and thinking there had been no more, bid him put them on again.
 And upon a certain time he gave it me in half a little white loaf,
 when we stood with the soldiers waiting to be called.
-And they seeing the book called '`The way to the kingdom,`' that it was printed at Amsterdam,
+And they seeing the book called [.book-title]#The Way to the Kingdom,# that it was printed at Amsterdam,
 asked me, '`What books I brought from there to Paltz, and how many?`'
 And one morning having sent for one of the priests.
-He got that written book of George Fox`'s of the Apostasy of Christendom,
+He got that written book of George Fox`'s of the [.book-title]#Apostasy of Christendom,#
```
Comment (maijapapaya) [inline history-of-quakers/modernized/28-appendix.adoc:739]: I wasn't sure if this was the actual title of the book...I tried to look it up, but couldn't find anything and this appendix isn't in the orginal PDF that I was using.

File: history-of-quakers/modernized/28-appendix.adoc (hunk 2 above)
Comment (jaredh159) [reply history-of-quakers/modernized/28-appendix.adoc:14]: Yeah, let's probably embed this.

File: history-of-quakers/modernized/28-appendix.adoc (hunk 3 above)
Comment (jaredh159) [reply history-of-quakers/modernized/28-appendix.adoc:531]: I think it's ok how it is. 👍

File: history-of-quakers/modernized/27-addenda.adoc [hunk 5]
```diff
@@ -21,6 +14,10 @@ And the original in Latin, soon after the first publication,
...
+--
+
+[.letter-heading]
 THE LIGHT UPON THE CANDLESTICK
```
Comment (jaredh159) [inline history-of-quakers/modernized/27-addenda.adoc]: Let's remove the all caps here (even though it looks the same when rendered), if we ever change the styling of the letter-heading, we'd want this to be normal-cased, I think.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Looks good, just a couple tweaks. 👍

File: history-of-quakers/modernized/28-appendix.adoc (hunk 2 above)
Comment (maijapapaya) [reply history-of-quakers/modernized/28-appendix.adoc:14]: I labeled it as a testimony....not sure if that's right!

**Review by jaredh159 — APPROVED** [review]

**Comment by jaredh159** [comment]
awesome job on a really big book! thanks Maija!

**Pull request merged**